UM/Qt/Bindings/Theme.py

```diff
@@ -116,10 +116,14 @@ def getSize(self, size) -> QSizeF:
         Logger.log("w", "No size %s defined in Theme", size)
         return QSizeF()
 
+    @pyqtSlot(str, str, result = "QUrl")
     @pyqtSlot(str, result = "QUrl")
-    def getIcon(self, icon_name: str) -> QUrl:
-        if icon_name in self._icons:
-            return self._icons[icon_name]
+    def getIcon(self, icon_name: str, px: str = "24") -> QUrl:
+        if px in self._icons:
+            if icon_name in self._icons[px]:
+                return self._icons[px][icon_name]
+            elif icon_name in self._icons["icons"]:  # Retrieve the "old" icon from the base icon folder
+                return self._icons["icons"][icon_name]
```

Comment:
I am not in favor of naming icons by their size (12, 24, 30 etc). I think it's best to use terms like small, default, medium, large and set the size using the sourceSize of the images.

Reply:
Automatically setting them to the source size, might be a bit overkill imo. I agree with the renaming to: small, default, large